.github/workflows/int-and-connected-test-run.yml

```diff
@@ -108,100 +132,81 @@ jobs:
-      - name: Deploy to cluster with Helm
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
-        with:
-          actions_subcommand: 'helmdeploy'
-          helm_create_secret_manager_secret_version: 0.0.8
-          helm_datarepo_api_chart_version: 0.0.739
-          helm_datarepo_ui_chart_version: 0.0.360
-          helm_gcloud_sqlproxy_chart_version: 0.19.13
-          helm_oidc_proxy_chart_version: 0.0.44
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: Fetch gitHash for deployed integration version
-        id: configuration
-        run: |
-          git_hash=$(git rev-parse --short HEAD)
-          echo "git_hash=${git_hash}" >> $GITHUB_OUTPUT
-          echo "Latest git hash for this branch: $git_hash"
-      - name: Wait for deployment to come back online
-        uses: broadinstitute/datarepo-actions/actions/wait-for-deployment@0.74.0
-        timeout-minutes: 20
-        env:
-          DESIRED_GITHASH: ${{ steps.configuration.outputs.git_hash }}
-          DEPLOYMENT_TYPE: 'api'
-      - name: Run test runner smoke tests via Gradle
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
-        env:
-          # unset the GitHub Action default JAVA_HOME to run with JDK 17
-          JAVA_HOME:
-          # output plain logs instead of json
-          TDR_LOG_APPENDER: 'Console-Standard'
-        with:
-          actions_subcommand: 'gradletestrunnersmoketest'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: Run integration tests via Gradle
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
+      - name: Run integration tests
         env:
           # required for azure tests
           AZURE_CREDENTIALS_APPLICATIONID: 0e29ec36-04e8-44d5-ae7c-50dc15135571
           AZURE_CREDENTIALS_HOMETENANTID: fad90753-2022-4456-9b0a-c7e5b934e408
           AZURE_CREDENTIALS_SECRET: ${{ secrets.AZURE_CREDENTIALS_SECRET }}
-          # unset the GitHub Action default JAVA_HOME to run with JDK 17
-          JAVA_HOME:
+          AZURE_SYNAPSE_SQLADMINUSER: ${{ secrets.AZURE_SYNAPSE_SQLADMINUSER }}
+          AZURE_SYNAPSE_SQLADMINPASSWORD: ${{ secrets.AZURE_SYNAPSE_SQLADMINPASSWORD }}
+          AZURE_SYNAPSE_WORKSPACENAME: tdr-snps-int-east-us-ondemand.sql.azuresynapse.net
+          # required for integration tests
+          GOOGLE_APPLICATION_CREDENTIALS: jade-dev-account.json
+          GOOGLE_SA_CERT: jade-dev-account.pem
+          IT_JADE_API_URL: http://localhost:8080
+          # postgres connection details
+          PGHOST: 127.0.0.1
+          PGPASSWORD: postgres
+          # required for integration tests
+          RBS_CLIENT_CREDENTIAL_FILE_PATH: rbs-tools-sa.json
+          RBS_INSTANCEURL: https://buffer.tools.integ.envs.broadinstitute.org
+          RBS_POOLID: datarepo_v1
           # output plain logs instead of json
           TDR_LOG_APPENDER: 'Console-Standard'
```

Comment:
From my comment on your other PR, I'd be in favor of moving this `TDR_LOG_APPENDER` setting to the top level for all tests (an exception to the new rule of narrowly scoping to env vars needed for a particular test type, which I like very much).

Reply:
I thought about this further, and on second thought I think it may be valuable to only keep it at the step level:

- a developer doesn't need to scroll back up to see if any environment variables are being inherited
- it makes it straightforward to know exactly which steps require `TDR_LOG_APPENDER`
- conversely, it also makes it clear which steps do NOT need `TDR_LOG_APPENDER`

so I am inclined to keep it as is.

Reply:
Fair enough!